nodejs-driver/cassandra.tests.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference path="cassandra.d.ts" />
+
+import * as cassandra from 'cassandra';
```

Comment:
documentation shows this as `'cassandra-driver'`. see https://github.com/datastax/nodejs-driver. is this the correct module name?